### ISSUE TITLE
SPF records not showing up

### ISSUE THREAD

**Issue by dexit** [Coopeh/PHP-MultiDig#2]
Websites with SPF records show that no SPF record are there.

**Comment by Coopeh** [comment]
Hi dExIT,

Unfortunately this is a problem with the NetDNS library or dig itself. If you choose TXT they will probably show. 

Cheers,
Ed

**Comment by dexit** [comment]
Hi Coopeh,
Its ok i worke dmy way through and fell asleep, bu ti can confirm that under TXT records it does show up, im no PHP guru but i was wondering of adding for CASE SPF a different way of filtering them pout by passing
`if(substr($source, 0, 6) == "v=spf1") {`
So it picks out the SPF record from all other TXT records :)

**Comment by Coopeh** [comment]
Yeah, sounds good. I'll add it in when I get chance or feel free to do a PR.

**Comment by dexit** [comment]
I have bigger plans for this ;)
ANd ill try my best, first testing testing testing...

**Comment by Coopeh** [comment]
Cool, looking forward to it :)
